cmd/icingadb-migrate/convert.go

```diff
@@ -701,30 +699,56 @@ func convertNotificationRows(
 //
 // [1]: https://github.com/Icinga/icinga2/blob/32c7f7730db154ba0dff5856a8985d125791c/lib/db_ido/dbevents.cpp#L1507-L1524
 // [2]: https://github.com/Icinga/icingadb/blob/8f31ac143875498797725adb9bfacf3d4/pkg/types/notification_type.go#L53-L61
-func convertNotificationType(notificationReason, state uint8) icingadbTypes.NotificationType {
+func convertNotificationType(notificationReason, state uint8) (string, error) {
 	switch notificationReason {
 	case 0: // state
 		if state == 0 {
-			return 64 // recovery
+			return "recovery", nil
 		} else {
-			return 32 // problem
+			return "problem", nil
 		}
-	case 1: // acknowledgement
-		return 16
-	case 2: // flapping start
-		return 128
-	case 3: // flapping end
-		return 256
-	case 5: // downtime start
-		return 1
-	case 6: // downtime end
-		return 2
-	case 7: // downtime removed
-		return 4
-	case 8: // custom
-		return 8
-	default: // bad notification type
-		return 0
+	case 1:
+		return "acknowledgement", nil
+	case 2:
+		return "flapping_start", nil
+	case 3:
+		return "flapping_end", nil
+	case 5:
+		return "downtime_start", nil
+	case 6:
+		return "downtime_end", nil
+	case 7:
+		return "downtime_removed", nil
+	case 8:
+		return "custom", nil
+	default:
+		return "", fmt.Errorf("bad notification type: %#v", notificationReason)
```

Comment:
Nitpick: The `%#v` format verb results in a lowercase hex representation for `uint8`. In the following two functions you have decided to use `%d` for the same type. Please keep them alike.

Reply:
Just copy pasted from the now removed `notification_type` code :)!